sdk/python/kfp/local/config.py

```diff
@@ -15,10 +15,14 @@
 import abc
 import dataclasses
 import os
-from typing import Union
+import tempfile
+from typing import Optional, Union
 
 from kfp import local
 
+# Default workspace path when no custom path is provided
+DEFAULT_WORKSPACE_PATH = '/tmp/kfp-workspace-default'
```

Comment:
I think rather than a default, could we just generate a temporary directory with `kfp-workspace-` as the prefix? Otherwise we can't run multiple local pipelines at the same time that use a workspace.